engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/views/content-access-mode-banner.html

```diff
@@ -1,5 +1,5 @@
 <div bst-alert="info" ng-show="contentAccessMode === 'org_environment'">
   <span translate>
-    Access to repositories is unrestricted in this organization. Hosts can consume all repositories available in the Content View they are registered to, regardless of subscription status.
+    This organization has Simple Content Access enabled. Access to repositories is unrestricted in this organization. Hosts can consume all repositories available in the Content View they are registered to, regardless of subscription status.
```

Comment:
Looking at it again, I think we can tighten this text up a little bit. Let me know what you think about:

`This organization has Simple Content Access enabled. Hosts can consume from all repositories in their Content View regardless of subscription status.`

I removed (imo) repetative 'this organization' and text around 'unrestricted' usage - because it *is* restricted to the content view.

Reply:
Lemme know what you think

Reply:
Updated